cms/static/sass/elements/_modal-window.scss

```diff
@@ -297,6 +297,11 @@
     .ui-loading {
       box-shadow: none;
     }
+
+    .modal-actions .action-move.is-disabled {
+      border: 1px solid $gray-l1 !important;
+      background: $gray-l1 !important;
+    }
```

Comment:
@mushtaqak Don't we have existing disabled styles in scss ? I believe we're already using disabled buttons in studio?